tck/src/main/resources/sql/derby/applicationidentity/schema.sql

```diff
@@ -407,6 +408,11 @@ CREATE TABLE employee_phoneno_type (
     TYPE VARCHAR(16) NOT NULL
 );
 
+CREATE TABLE employee_language (
```

Comment:
Use plural? 'employee_languages`?

Reply:
I renamed the join table to employee_languages